### PR TITLE
kie-issues#212: Remove DMN Guided Tour

### LINKED ISSUE #212: Remove DMN Guided Tour from KIE Sandbox
# Context
This issue is about removal of the component below:
Screenshot 2023-04-20 091359

It contains three options
## Skip tour and start DMN Runner
This option brings user to this popup below:
Screenshot 2023-04-20 091935

This popup is not accessible from other application menus or buttons. The question is if we should simplify DMN Guided tour popup from the first screenshot to contain just **Start DMN Runner** option or we should remove also **Setup DMN Runner** popup as part of kiegroup/kie-issues/issues/212

## Skip tour
This option simply closes the guided tour popup, there is no action item to take into account

## Take tour
This option brings user to wizard with few steps like below:
Screenshot 2023-04-20 092717

So we need to be sure we remove source code of
- [ ] wizard pages views
- [ ] logic of communicating between wizard react component and gwt events executed in canvas of the diagram

### PR DESCRIPTION
**Pull request by jomarko**
This PR closes https://github.com/kiegroup/kie-issues/issues/212

### CODE REVIEW

**Review by yesamer — APPROVED** [review]
Thanks @jomarko

File: packages/editor/tests/embedded/stateControl/Hooks.test.ts [hunk 1]
```diff
@@ -37,8 +37,6 @@ describe("useDirtyState", () => {
       setContent: jest.fn(),
       validate: jest.fn(),
       setTheme: jest.fn(),
-      getElementPosition: jest.fn(),
-      onKeyDown: jest.fn(),
```
Comment (yesamer) [inline packages/editor/tests/embedded/stateControl/Hooks.test.ts:41]: Is this change intentional? @jomarko

**Review by yesamer — APPROVED** [review]

File: packages/editor/tests/embedded/stateControl/Hooks.test.ts (hunk 1 above)
Comment (jomarko) [reply packages/editor/tests/embedded/stateControl/Hooks.test.ts:41]: I think I wrongly resolved conflicts, I hope I fixed with https://github.com/kiegroup/kie-tools/pull/1576/commits/e6e62979818cc6622808e28893a3afc4aa0f3bfd

**Review by tiagobento — APPROVED** [review]

**Comment by yesamer** [comment]
@jomarko Can you please solve the above conflicts? ^

**Comment by jomarko** [comment]
re https://github.com/kiegroup/kie-tools/pull/1576#issuecomment-1527476963 ,

pushed, lets wait for CI

**Comment by yesamer** [comment]
@jomarko  `ERR_PNPM_OUTDATED_LOCKFILE  Cannot install with "frozen-lockfile" because pnpm-lock.yaml is not up to date with packages\yard-vscode-extension\package.json`
